### DISCUSSION TITLE
Weekly Governance Update (6 May 2022)

### DISCUSSION THREAD

**Discussion by kaitlin-beegle** [filecoin-project/FIPs#370] [Announcements]
*This weekly report provides a high-level overview of what's happening with Filecoin governance.* 

*Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.*

*If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](filecoinproject.slack.com).*

---
**Accepted FIPs** - FIPs awaiting network implementation
* FIP0027 - Change type of DealProposal Label field from a (Golang) String to a Union [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0027.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/262)
* FIP0029 - Beneficiary address for storage providers [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0029.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/253)

**Last Call FIPs** - final opportunity for community feedback! 
* N/A

**Current FIP Drafts** - *now accepting community feedback; merged into repo*
* FIP0030 - Introducing the Filecoin Virtual Machine [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0030.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/287)
* FIP0031 - Atomic switch to non-programmable FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0031.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/296)
* FIP0032 - Gas model adjustment for non-programmable FVM [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
* FIP0033 - Explicit premium for FIL+ Verified deals [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0033.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/243)

**Open Pull Requests** - *upcoming FIP drafts awaiting technical review; now accepting community feedback*
* Indexer FIP  [(PR)](https://github.com/filecoin-project/FIPs/pull/341)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/337)
* Add FIP proposing p256 address types  [(PR)](https://github.com/filecoin-project/FIPs/pull/321)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/346)
* FIP gas model adjustment for user programmability [(PR)](https://github.com/filecoin-project/FIPs/pull/317)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
* Set pre-commit deposit independent of sector content [(PR)](https://github.com/filecoin-project/FIPs/pull/307)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/290)
* Supporting contracts as deal clients in the built-in storage market actor [(PR)](https://github.com/filecoin-project/FIPs/pull/305)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/271)
* FIP Filecoin Message Replay Protection [(PR)](https://github.com/filecoin-project/FIPs/pull/300)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
* Askv2 Protocol [(PR)](https://github.com/filecoin-project/FIPs/pull/224)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/223)

**🔥 HOT 🔥 Discussion Topics**
* [Proposal: FIP on Indexer](https://github.com/filecoin-project/FIPs/discussions/337) - This discussion topic, the latest in the FIPs repo, proposes an 'advertising' protocol for FIP indexer nodes. This is an exciting proposal that will make it easier for the Filecoin indexer to remain up-to-date with network data.  For more information, check out the FIP draft, or revisit [Core Devs #41](https://github.com/filecoin-project/tpm), during which the team presented their proposal to network implementers. 

**Other Governance Updates**
* The v16 'Skyr' Network is currently in progress.  To keep track of timeline changes and other updates, be sure to keep tabs on the [network planning thread](https://github.com/filecoin-project/tpm/discussions/90) in the [TPM repo](https://github.com/filecoin-project/tpm).

**Comment by anorth** [comment]
FIP0033 is deferred, so I don't think needs to be listed as "current".
